raiden/tests/unit/test_transfer.py

```diff
@@ -187,15 +187,23 @@ def test_mediated_transfer(raiden_network):
 
     amount = 10
 
-    alice_app.raiden.api.transfer(
+    result = alice_app.raiden.api.transfer_async(
         token_address,
         amount,
         charlie_address,
     )
 
-    gevent.sleep(1.)
+    assert channel_ab.locked == amount
+
+    # cannot do these assertions because the message has not been processed yet
+    # assert channel_ba.outstanding == amount
```

Comment:
When reading the comment it sounds like a TODO to me. Perhaps try to rephrase? I don't think it needs to be removed, it serves an informational purpose.